pyproject.toml

```diff
@@ -0,0 +1,109 @@
+[tool.poetry]
+name = "kiwi"
+version = "10.0.0"
+description = "KIWI - Appliance Builder"
+license = "GPL-3.0-or-later"
+readme = "README.rst"
+homepage = "https://osinside.github.io/kiwi/"
+documentation = "https://osinside.github.io/kiwi/"
+repository = "https://github.com/OSInside/kiwi"
+
+authors = [
+   "Marcus Schäfer <marcus.schaefer@suse.com>",
+]
+maintainers = [
+   "Marcus Schäfer <marcus.schaefer@suse.com>",
+   "David Cassany Viladomat <dcassany@suse.com>",
+   "Neal Gompa <neal@gompa.dev>",
+   "Dan Čermák <dcermak@suse.com>",
+]
+
+packages = [
+   { include = "kiwi"},
+]
+
+include = [
+   { path = ".bumpversion.cfg", format = "sdist" },
+   { path = ".coverage*", format = "sdist" },
+   { path = ".virtualenv.requirements*.txt", format = "sdist" },
+   { path = "doc/source", format = "sdist" },
+   { path = "doc/Makefile", format = "sdist" },
+   { path = "dracut", format = "sdist" },
+   { path = "helper", format = "sdist" },
+   { path = "kiwi.yml", format = "sdist" },
+   { path = "Makefile", format = "sdist" },
+   { path = "package", format = "sdist" },
+   { path = "test", format = "sdist" },
+   { path = "tox.ini", format = "sdist" },
+]
+
+classifiers = [
+   # classifier: http://pypi.python.org/pypi?%3Aaction=list_classifiers
+   "Development Status :: 5 - Production/Stable",
+   "Intended Audience :: Developers",
+   "Operating System :: POSIX :: Linux",
+   "Topic :: System :: Operating System",
+]
+
+[tool.poetry.urls]
+"Bug Tracker" = "https://github.com/OSInside/kiwi/issues"
+
+[tool.poetry.dependencies]
+python = "^3.9"
+docopt = ">=0.6.2"
+lxml = ">=4.6.0"
+requests = ">=2.25.0"
+PyYAML = ">=5.4.0"
+setuptools = ">=50"
+simplejson = ">=3.17.0"
+
+# Optional dependencies for markup extra
+anymarkup-core = { version = ">=0.8.0", optional = true }
+xmltodict = { version = ">=0.12.0", optional = true }
```

Comment:
Shouldn't these be put into a separate group?

Reply:
No, because I wanted them as a runtime extra.